test/full_coverage_plugin.py

```diff
@@ -59,6 +60,12 @@ def pytest_runtestloop(session):
     if os.name == 'nt':
         cov.exclude('pragma: windows no cover')
 
+    if sys.platform == 'darwin':
+        cov.exclude('pragma: osx no cover')
+
+    if 'OPENSSL_OLD' in os.environ:
+        cov.exclude('pragma: alpn no cover')
```

Comment:
These patterns use a name to indicate *when* the code is not being covered.
Following this idea, `alpn no cover` would mean, code is not covered when ALPN is present.
I think a better name would be `pragma: missing-alpn no cover` or `pragma: openssl-old no cover`.